commons-jcs-jcache/src/main/java/org/apache/commons/jcs3/jcache/cdi/CacheResolverFactoryImpl.java

```diff
@@ -50,7 +50,7 @@ public CacheResolver getCacheResolver(final CacheMethodDetails<? extends Annotat
     public CacheResolver getExceptionCacheResolver(final CacheMethodDetails<CacheResult> cacheMethodDetails)
     {
         final String exceptionCacheName = cacheMethodDetails.getCacheAnnotation().exceptionCacheName();
-        if (exceptionCacheName == null || exceptionCacheName.isEmpty())
+        if (exceptionCacheName.isEmpty())
```

Comment:
this is likely a false positive, can be null

Reply:
indeed,
TY